PlaywrightEvershop/PageObjects/EverShopCartPage.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.Playwright;
+
+namespace PlaywrightEverShop.PageObjects;
+
+public class EverShopCartPage(IPage page)
+{
+    // Locators
+    private ILocator ViewCartButton => page.Locator(".add-cart-popup-button");
+    private ILocator CheckoutButton => page.Locator("//span[contains(text(),'CHECKOUT')]");
+    
+    // Methods
+    public async Task GoToCartPage()
+    {
+        await ViewCartButton.ClickAsync();
+    }
+    
+    public async Task MakeCheckout()
+    {
+        await CheckoutButton.ClickAsync();
+    }
+
+    public async Task<string> GetProductName()
+    {
+        var productCard = await page.QuerySelectorAsync(".product-info");
+        var productName = await productCard!.InnerTextAsync();
+        
+        return ExtractProductName(productName).ToLower();
+    }
+    
+    private static string ExtractProductName(string productName)
+    {
+        return productName.Split("\n")[0];
+    }
```

Comment:
It is good to make new functions for reusable code, but no need to create function with 1 line which most likely will not be re-used by other code and it is actually closely related to the function where it is called from.
I would implement this like
    public async Task<string> GetProductName()
    {
        var productName = await page.QuerySelectorAsync(".product-info").InnerTextAsync();
        return productName.Split("\n")[0];
    }